client/src/containers/Test/Test.tsx

```diff
@@ -1,10 +1,26 @@
 import React from 'react';
 import { AnimatedH1 } from './styled';
+import { Button, ButtonA } from '../../components/UI/Button';
 
 const Test = () => (
```

Comment:
Maybe better will be used storybook library to show different variant of component

Reply:
i didn't know about this library, ty for sharing i will check that